go/oasis-test-runner/oasis/oasis.go

```diff
@@ -252,8 +254,8 @@ type NetworkCfg struct { // nolint: maligned
 	// created in this test network.
 	DefaultLogWatcherHandlerFactories []log.WatcherHandlerFactory `json:"-"`
 
-	// UseAbstractGrpcSockets will use abstract sockets for internal GRPC communication.
-	UseAbstractGrpcSockets bool
+	// UseCustomGrpcSocketPath contains whether nodes should use internal.sock in datadir or test-runner should generate custom (shorter) ones.
+	UseCustomGrpcSocketPath bool
```

Comment:
Missing codec annotations. If you don't want this to be configurable in JSON fixtures, set it as `json:"-"`.